test/node/src/camel-case.test.ts

```diff
@@ -203,5 +209,33 @@ for (const dialect of BUILT_IN_DIALECTS) {
         },
       })
     })
+
+    // Can't run this test on SQLite because json field returns string
+    if (dialect !== 'sqlite') {
```

Comment:
could we cover sqlite as well? it's a `JSON.parse` or a an assertion against a string away..